iron/distribution.yaml

```diff
@@ -3892,6 +3892,12 @@ repositories:
       url: https://github.com/nerian-vision/nerian_stereo_ros2.git
       version: default
     status: developed
+  netft_utils:
+    source:
+      type: git
+      url: https://github.com/tamu-edu/rad_lab_netft_utils.git
```

Comment:
There are 2 main problems I can see with this repository:

1.  It is a fork of another repository.  While that doesn't automatically disqualify it from being listed here, we typically prefer to use the upstream where possible.  Can you describe the reason for the fork, and the relationship between this repository and the one it is forked from?
2.  The licensing of it is unclear.  In particular, https://github.com/tamu-edu/rad_lab_netft_utils/blob/main/netft_utils/License.txt uses similar text to a BSD 3-clause, but is not exactly the same.  Additionally, https://github.com/tamu-edu/rad_lab_netft_utils/blob/f5fd670aaf0a0ffa4054334cbc1f86ff16b99219/netft_utils/package.xml#L15 doesn't clearly state the license.

We can possibly work through that first issue, but because the license is not approved by the OSI, we actually aren't qualified to determine whether we can distribute this or not.  Is there anyway those licenses can be made standard 3-clause BSD?

Reply:
The reason for the fork is that my lab is maintaining the ROS2 port of the package, however we have pushed the changes to the upstream repository. I updated the pull request to reflect this change.
We have contacted the original authors of the package about the license, and I will let you know if anything changes.

Reply:
Hi, I represent the authors of the upstream repo. I don't know that we can change the license since Los Alamos specified this one for work they have funded. It is, in all practical ways, identical to BSD-3 however. I could reach our to our Los Alamos contact and ask about this.

Reply:
I understand that this license is what Los Alamos National Labs will use for all software approved for open source release (https://www.lanl.gov/software/open-source-software.php). Would it be possible to get this license added to the approved list? They have several groups increasingly working in robotics so this may be the easier path.

Reply:
> I understand that this license is what Los Alamos National Labs will use for all software approved for open source release (https://www.lanl.gov/software/open-source-software.php). Would it be possible to get this license added to the approved list? They have several groups increasingly working in robotics so this may be the easier path.

Unfortunately, we really have no way of doing this.  We aren't lawyers, so we don't know if this license allows us to legally redistribute it.

Reply:
That makes sense. I have reached out to LANL to see if we can put the license through the OSI approval process (https://opensource.org/licenses/review-process). I have not done that before, but it looks like it takes 60 days.

Reply:
An update: I submitted the license to OSI and they said it should be easily approved. Now just waiting for that to be official